package.json

```diff
@@ -88,8 +88,8 @@
     "compression": "^1.6.0",
     "express": "^4.13.3",
     "express-session": "^1.12.1",
-    "history": "1.17.0",
     "file-loader": "^0.8.5",
+    "history": "2.0.0-rc2",
```

Comment:
we need it : https://github.com/erikras/react-redux-universal-hot-example/pull/833/files#diff-e6a5b42b2f7a26c840607370aed5301aR17